src/components/nav/NavBar.vue

```diff
@@ -1,14 +1,60 @@
 <template>
   <div id="nav">
-    <router-link to="/" class="nav-home-link">
-      <img src="../../assets/c4c-banner-logo.png" class="home-nav" />
+    <div class="big-menu">
+      <router-link to="/" class="nav-home-link">
+        <img src="../../assets/c4c-banner-logo.png" class="home-nav" />
+      </router-link>
+      <router-link to="/news" class="nav-item">News</router-link>
+      <router-link to="/events" class="nav-item">Events</router-link>
+      <router-link to="/board" class="nav-item">Board</router-link>
+      <router-link to="/apply" class="nav-item">Apply</router-link>
+      <router-link to="/auth" class="nav-item">Sign In</router-link>
+    </div>
+    <img
+      class="hamburger-icon"
+      @click="toggle"
+      src="../../assets/hamburger.png"
+    />
+    <router-link @click.native="toggle" to="/">
```

Comment:
I don't think we should bind the `toggle` to the logo. it should just take people to homepage. what do you think?

Reply:
Oops. I did this so that the menu disappears if you click the logo with the menu out, but I didn't consider that it would also make the menu appear if you click on the logo when the menu isn't out. I'll fix this